lib/backend/repositories/auth_repo.dart

```diff
@@ -129,4 +129,9 @@ class AuthRepo {
     themesProvider.setTheme("rainbow");
     await authService.logoutUser();
   }
+
+  // Delete user account
+  Future<void> deleteUserAccount(String userAddress, String password) async {
```

Comment:
Think we should add some assertion checks here to ensure the userAddress and password are not null or empty.